src/process/process/libs/scheduler/scheduler.hpp

```diff
@@ -7,75 +7,76 @@
 
 #include "logging.hpp"
 
-namespace scheduler {
+namespace process {
```

Comment:
as mentioned in scheduler.cpp, I would prefer scheduler to not be in the process namespace